### PR TITLE
Add filter to decorate headers to support CORS.

### PR DESCRIPTION
**Pull request by melowe**

### CODE REVIEW

File: jaxrs-service/src/main/java/com/quorum/tessera/api/filter/DomainFilter.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (prd-fox) [inline jaxrs-service/src/main/java/com/quorum/tessera/api/filter/DomainFilter.java]: Remove extra newline.

File: jaxrs-service/src/test/java/com/quorum/tessera/api/filter/DomainResponseFilterTest.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        when(requestContext.getHeaderString("Origin")).thenReturn(null);
+
+        domainResponseFilter.filter(requestContext, responseContext);
+
+        verify(requestContext).getUriInfo();
+        verify(requestContext).getHeaderString("Origin");
+
+    }
+    
+        @Test
```
Comment (prd-fox) [inline jaxrs-service/src/test/java/com/quorum/tessera/api/filter/DomainResponseFilterTest.java]: Fix formatting

**Review by prd-fox — DISMISSED** [review]
Was there a reason why we are only adding this to particular resources instead of all of them?

**Review by nicolae-leonte-go — DISMISSED** [review]

**Review by namtruong — DISMISSED** [review]

**Comment by Krish1979** [comment]
https://github.com/jpmorganchase/tessera/issues/709

**Comment by Krish1979** [comment]
[https://github.com/jpmorganchase/tessera/issues/709](url)

**Review by namtruong — APPROVED** [review]

**Comment by prd-fox** [comment]
Linked issue: https://github.com/jpmorganchase/tessera/issues/709

**Pull request merged**